numba/tests/support.py

```diff
@@ -107,6 +107,8 @@
                                           "parameter area' in "
                                           "LowerCall_64SVR4"))
 
+skip_m1 = unittest.skipIf(config.IS_OSX_ARM64, "Unreliable on OSX arm64")
```

Comment:
This seems like quite a generic name for a function that is skipping for a specific reason? Suggest either making this `skip_m1` take the "reason" string such that it is generic, or making this function name a lot more specific.

Reply:
I will make the skip name more specific.